functional-tests/src/main/java/com/redhat/che/selenium/core/client/RhCheTestWorkspaceServiceClient.java

```diff
@@ -73,8 +73,18 @@ public RhCheTestWorkspaceServiceClient(
     this.token = this.owner.obtainAuthToken();
   }
 
-  public Workspace createWorkspaceWithCheStarter() throws Exception {
-    return createWorkspace(null, 0, null, null);
+  public Workspace createWorkspaceWithCheStarter(String templateFileName) throws Exception {
+    if (owner == null) {
+      throw new IllegalStateException("Workspace does not have an owner.");
+    }
+    this.cheStarterWrapper.checkIsRunning(this.token);
+    String name =
+        this.cheStarterWrapper.createWorkspace(
+            "/requests/workspace/osio/" + templateFileName, token);
+    return requestFactory
+        .fromUrl(getNameBasedUrl(name, owner.getName()))
+        .request()
+        .asDto(WorkspaceDto.class);
```

Comment:
This whole method seems to me like a duplication of code (see method `createWorkspace(String, int, MemoryMeasure, WorkspaceConfigDto)`)

Reply:
This method allows us to use a template. `createWorkspace` doesn't take a template as a parameter (because it overrites super class method) and it uses a vert.x stack by default.